### DISCUSSION TITLE
How to cancel "AUTO" during battle?

### DISCUSSION THREAD

**Discussion by HubKing** [ihhub/fheroes2#6996] [Q&A]
When I almost lost, I want to run away instead of being killed. But doing all the battle manually requires a lot of efforts, so I want to use Auto. The problem is that I don't know what is the exact way to cancel "Auto". The button at the bottom-left has no feedback. That is, I don't know if it is auto or not. Sometimes if I keep clicking it it gets cancelled, but sometimes it doesn't. How to cancel auto mode? And shouldn't the button be toggled or something (some sort of visual indicator that Auto is active)?

**Comment by oleg-derevenetz** [comment]
You can cancel auto battle by pressing on that button, `A` key or `Esc` key **_during the turn of any of your units_**.

**Comment by LeHerosInconnu** [comment]
Hello everyone,

Yes, it's true that this is a problem.
I made a proposal to animate the button and play a sound to inform the player that his actions are taken into account here: https://github.com/ihhub/fheroes2/discussions/4385.
